force-app/main/default/classes/ServiceSessionSelector.cls

```diff
@@ -44,10 +47,34 @@ public with sharing class ServiceSessionSelector {
             String.valueOf(ServiceSession__c.Status__c) + ' IN :allowedSessionStatuses'
         );
 
+        System.debug('allowedSessionStatuses : ' + allowedSessionStatuses);
         queryBuilder.withOrderBy(String.valueOf(ServiceSession__c.SessionStart__c));
 
+        System.debug(queryBuilder.buildSoqlQuery());
         allSessions = Database.query(queryBuilder.buildSoqlQuery());
 
         return Security.stripInaccessible(AccessType.READABLE, allSessions).getRecords();
     }
+
+    public Set<String> getServiceSessionStatusBuckets() {
```

Comment:
This is another method that I would suggest gets a re-name... it's returning statuses, not buckets

Reply:
And actually - Jenny should maybe have the final word here - but I don't know if this method needs to exist in the ServiceSessionSelector. The way I think of it, this method belongs more in the ServiceSessionService - or even the controller, because the logic coded in here (returning status values for the two buckets that get displayed on the home view) is more specific to the front end. I don't think there needs to be a separate selector method for this behavior, just calling straight through to the FieldBucketSelector from the ServiceSessionService....

Reply:
@bmuniswamy I still believe this method needs to be renamed to reflect that it's returning a set of status values, not buckets - I would go with "getAllowedServiceSessionStatusValues()" maybe? And then update the line where you are calling it (line #18?) with the new name